terraform/modules/tf_stream_alert_globals/main.tf

```diff
@@ -4,3 +4,26 @@ module "alerts_firehose" {
   prefix     = "${var.prefix}"
   region     = "${var.region}"
 }
+
+// TODO: Autoscaling
+resource "aws_dynamodb_table" "alerts_table" {
+  name           = "${var.prefix}_streamalert_alerts"
+  read_capacity  = "${var.alerts_table_read_capacity}"
+  write_capacity = "${var.alerts_table_write_capacity}"
+  hash_key       = "RuleName"
+  range_key      = "Timestamp"
+
+  // Only the hash key and range key attributes need to be defined here.
+
+  attribute {
+    name = "RuleName"
+    type = "S"
+  }
+  attribute {
+    name = "Timestamp"
+    type = "S"
```

Comment:
Is this intended to be in `Date/Calendar` format as specified [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html)?

Reply:
Yes, but `Date` is not actually a Dynamo type. It's really [just a String](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html):

_You can use the string data type to represent a date or a time stamp. One way to do this is by using ISO 8601 strings_

You can confirm this by looking at the Dynamo UI as well:
![screen shot 2018-02-27 at 4 23 38 pm](https://user-images.githubusercontent.com/3608925/36762907-9e2a27da-1bda-11e8-910e-ace8e9b7bbd9.png)

No `Date`, only `String`. I chose ISO 8601 so it was easy to sort and could be loaded by other applications as needed